app.py

```diff
@@ -86,3 +101,36 @@ def proba(hour):
     p = (p_precip + p_wind * 2) / 3
 
     return gradient((1 - p) * 100, max=100, end="green")
+
+
+@app.template_filter("localized_condition")
+def localized_condition(code):
+    """Translate weather condition from code"""
+    
+    # conditions list from https://www.weatherapi.com/docs/#weather-icons
+    c_file = open('translations/conditions.json')
+    c_data = json.loads(c_file.read())
+
+    condition = next((item for item in c_data if item['code'] == code), None)
+
+    for lang in condition['languages']:
+        if lang['lang_iso'] == get_locale():
+            localized_condition_text = lang['day_text']
+            break
+    else:
+        localized_condition_text = condition['day']
+
+    return localized_condition_text
+
+
+@app.template_filter("localized_azimuth")
+def localized_azimuth(code):
+    """Translate azimuth from code"""
+    
+    # conditions list from https://www.weatherapi.com/docs/#weather-icons
+    a_file = open('translations/azimuths.json')
+    a_data = json.loads(a_file.read())
+
+    azimuth = a_data[code][get_locale()]
```

Comment:
In this method, you seem to be sure you'll find `get_locale()` in the translations, whereas in `localized_condition` you don't (fallback on default text). Why is that?

Reply:
(I really must have missed something because `azimuths.json` only has `en` and `fr` translations 🤔)

Reply:
Fixed in 47547369c269e262a564ac121b7d455c50d07cbb, using `accept_language.best_match()` `default` argument (https://github.com/pallets/werkzeug/blob/2dd99eb3cf9c7dd1ece27e36a24242bb02d340ef/src/werkzeug/datastructures.py#L1918)